docker/docker-compose.full.yml

```diff
@@ -0,0 +1,41 @@
+services:
+  path_gateway:
+    container_name: path_gateway
+    build:
+      context: ..
+      args:
+        IMAGE_TAG: ${IMAGE_TAG:-development}
+    ports:
+      # TODO_IMPROVE: Update the port to match the actual value defined in the config yaml
```

Comment:
Can you add more detail on what you mean here?

Reply:
The port is a configurable value so ideally we find a way to dynamically set the exposed port in the docker compose from the value set in the config YAML.

Reply:
Updated comment.